packages/db/src/pouch/databases.ts

```diff
@@ -152,6 +153,24 @@ export abstract class Databases<C extends Collections> implements Workspace<C> {
     }
   }
 
+  public async merge<N extends CollectionName<C>>(
```

Comment:
This can be `private` I think - we don't need to call it elsewhere, right?

Reply:
Hm, I thought about that but I can't add the `private` modifier on the interface in `pouch/types`. Any thoughts?